packages/application-generic/src/services/in-memory-provider/cache-in-memory-provider.service.ts

```diff
@@ -93,10 +106,7 @@ export class CacheInMemoryProviderService {
   }
 
   public providerInUseIsInClusterMode(): boolean {
-    const providerConfigured =
-      this.inMemoryProviderService.getProvider.configured;
-
-    return this.isCluster || providerConfigured !== InMemoryProviderEnum.REDIS;
+    return this.loadedProvider.provider !== InMemoryProviderEnum.REDIS;
```

Comment:
For the cases that providers are misconfigured and it fell backs to other providers we can't rely on if the cluster mode has been set or not.